src/main/java/Model/Database/Tables/Table/T_Project.java

```diff
@@ -23,9 +23,9 @@
 
     // Atributes
     private int a_pk;
-    private String a_name;
-    private Date a_created;
-    private Date a_deleted;
+    private String a_Name;
+    private Date a_CreatedAt;
+    private Date a_DeletedAt;
```

Comment:
niekde by sa mohol dat jeden komentar na tuto konvenciu, ci to netreba? dame toto do dokumentacie niekde?